features/step_definitions/sign-in_steps.rb

```diff
@@ -28,4 +28,46 @@
 And /I enter correct email and password for a not confirmed patient/ do
   fill_in "user[email]", with: "notconfirmed@test.com"
   fill_in "user[password]", with: "Test123"
+end
+
+When(/^I visit home page$/) do
+  visit '/'
+end
+
+When(/^I visit survey page$/) do
```

Comment:
We can probably combine lines 37 and 41 into one, but not needed.

Reply:
We can do that but then we may use "When I visit ... page" for other purposes that has different route pattern (such as forgot password page) which will cause problem. I say let's keep it this way for now.